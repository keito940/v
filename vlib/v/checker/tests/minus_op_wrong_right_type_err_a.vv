struct Aaa{}
fn main() {
    10 - Aaa{}
}
