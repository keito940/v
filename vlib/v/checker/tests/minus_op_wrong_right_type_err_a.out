vlib/v/checker/tests/minus_op_wrong_right_type_err_a.v:3:10: error: mismatched types `int` and `A`
    1| struct A{}
    2| fn main() {
    3|     10 - A{}
                ~~~
    4| }
